.github/workflows/meta-package.yml

```diff
@@ -9,8 +9,8 @@ jobs:
     name: Sync
     runs-on: ubuntu-latest
     outputs:
-      tags-matrix: steps.tags-matrix.outputs.result
-      latest-release: steps.latest-release.outputs.result
+      tags-matrix: ${{ steps.tags-matrix.outputs.result }}
+      latest-release: ${{ steps.latest-release.outputs.result }}
```

Comment:
That was probably my most dump and so long to figure out mistake of the last year.

Reply:
GitHub Actions: making you feel dumb since 2020 ™️